custom_components/tahoma/config_flow.py

```diff
@@ -49,6 +65,15 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
 
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry):
+        return ThermoOptionsFlowHandler(config_entry)
```

Comment:
Can this be options based on the devices? or not possible here yet

Reply:
Not that I know of...